src/app.tsx

```diff
@@ -16,6 +16,9 @@ import {
   color_canvas_secondary_inverted,
   color_accent_secondary,
   color_brand_quinary,
+  validate_tile_ok_dark,
+  validate_tile_warning_dark,
```

Comment:
Please rename it in more common way. Try to abstract from particular coloring component names.
Follow the exiting naming convention.